traefik/tests/traefik-config_test.yaml

```diff
@@ -431,6 +451,22 @@ tests:
       - failedTemplate:
           errorMessage: "ERROR: local plugin local-demo is missing moduleName !"
 
+  - it: should allow useUnsafe option for localPlugins
```

Comment:
Wdyt of adding unsafe test L392 in the already existing localPlugins test instead of adding a new one ? (to test with unsafe applied only on one of the plugins)